packages/components/src/components/wallet-card/wallet-card.tsx

```diff
@@ -9,9 +9,16 @@ import { WalletIcon } from '../wallet-icon';
 import './wallet-card.scss';
 import classNames from 'classnames';
 
-type TWalletCardProps = {
+export type TWalletCardProps = {
```

Comment:
```suggestion
type TWalletCardProps = {
```
@aum-deriv Keep the prop type of a component isolated to the component itself.